src/app/modules/main/module.nim

```diff
@@ -534,7 +537,11 @@ method communityJoined*[T](
   self.communitySectionsModule[community.id].load(events, settingsService, contactsService, chatService, communityService, messageService, gifService)
 
   let communitySectionItem = self.createCommunityItem(community)
-  self.view.addItem(communitySectionItem)
+  if (firstCommunityJoined):
+    # If there are no other communities, add the first community after the Chat section in the model so that the order is respected
+    self.view.model().addItem(communitySectionItem, self.view.model().getItemIndex(conf.CHAT_SECTION_ID) + 1)
+  else:
+    self.view.model().addItem(communitySectionItem)
```

Comment:
I'm not too familiar with the model creation for this part.. How exactly is ensured that the **next** community item (after the first one) is then also put in the correct position?
The correct position being: After the first/previous community item, but before the wallet/browser/settings item?

Reply:
@PascalPrecht thanks for mention here. I've checked this so... 

> I'm not too familiar with the model creation for this part.. How exactly is ensured that the next community item (after the first one) is then also put in the correct position?

`StatusAppNavBar` component which is a part of `StatusQ` knows about 2 components (but both of them are actually `StatusNavBarTabButton`, the difference is only that the community button has a context menu) which are used for displaying "regular" and "community" button, and also it knows about `sectionModel` which contains a list of items (buttons) to display:
```
property var sectionModel: []
property Component regularNavBarButton
property Component communityNavBarButton
```
Now `StatusAppNavBarFilterModel` component with `id: navBarModel` (child component of `StatusAppNavBar` component) actually goes through the provided `sectionModel` and adds appropriate "regular" button to the `StatusAppNavBar` and the first time it encounters to the "community" button there is another `StatusAppNavBarFilterModel` with `id: navBarCommunityModel` which filters all communities from the `sectionModel` and display them one after another, and after that `StatusAppNavBarFilterModel` component with `id: navBarModel` continues with the rest of "regular" buttons.

And that's the key why the first and all community after it were added to the end of the list. Placing only the first community at the index 1 in the `sectionModel` regardless what is the index of other communities will be handled properly on the qml side.